internal/v1/db/erc20.go

```diff
@@ -34,25 +41,25 @@ func RedisGetERC20Balance(contract string, address string) (string, error) {
 }
 
 func RedisSetERC20TokensDirectory(result string) {
-	err := rdb.Set(ctxRedis, erc20TokensDirectoryKey, result, time.Duration(oneDayExpiration*int(time.Second))).Err()
+	err := rdb.Set(ctxRedis, getErc20TokensDirectoryKey(), result, time.Duration(oneDayExpiration*int(time.Second))).Err()
```

Comment:
Just for the code eto be more readable, can you create a variable of the key first ?

Reply:
Another optionis to have this instantiated in an init function so you dont have to call it on every function